src/store/initialState.js

```diff
@@ -10,6 +10,13 @@ const initialState = {
       error: ''
     },
     isAuthenticated: false
+  },
+  comments: {
+    fetchingArticleComments: false,
+    fetchingCommentsError: '',
+    currentArticleComments: [],
+    postingNewComment: false,
+    newCommentError: ''
```

Comment:
couldn't `postingNewComment` and `fetchingArticleComments` just be `loading`, and `newCommentError` and `fetchingCommentsError` as `error`.  This will also lead to reducing the `createCommentRequest()`, `getArticleCommentsLoading()`, `getArticleCommentsError()` `createCommentError()` functions from actions to something like `setLoading()` and `setError()`

Reply:
Yeah. I agree. 

However, I did it this way because I needed to handle error states for when the fetching comments, errors out and also for when posting a new comment, errors out. This is a bit more explicit as well.